pyecharts/charts/composite_charts/timeline.py

```diff
@@ -29,7 +29,11 @@ def add_schema(
         pos_right: Optional[str] = None,
         pos_top: Optional[str] = None,
         pos_bottom: Optional[str] = "-5px",
+        label_opts: Optional[opts.LabelOpts] = None,
     ):
+        if isinstance(label_opts, opts.LabelOpts):

```

Comment:
This is a code smell when isinstance has to be used. but this is not a bug but a design problem.